src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -84,6 +84,18 @@
       <TestCopyLocal Include="@(OverridenRuntimeContent)" />
      </ItemGroup>
 
+    <ItemGroup Condition="'$(CoreLibRoot)' != ''">
```

Comment:
You included this in a PR related to coverage, but this isn't specific to coverage, right?